Source/DataSources/KmlDataSource.js

```diff
@@ -2520,6 +2528,8 @@ define([
             up : defined(camera) ? Cartesian3.clone(camera.upWC) : undefined,
             bbox : defined(camera) ? camera.computeViewRectangle() : Rectangle.clone(Rectangle.MAX_VALUE)
         };
+
+        this._ellipsoid = options.ellipsoid;
```

Comment:
Yes, regarding my previous comment, right here use `defaultValue`, then use that value throughout.